### ISSUE TITLE
`@image-url()` with URL working in SlintPad produces insubstantial error message in local build, and is silent in preview

### ISSUE THREAD

**Issue by Enyium** [slint-ui/slint#6412]
The docs page ["Syntax > Types > Images"](https://docs.slint.dev/1.8.0/docs/slint/src/language/syntax/types#images) has an example that works in SlintPad when you click "Edit" below the example. But when I copy it under Windows 10 into a Rust project with Slint (not matter whether FemtoVG or Skia), it produces the following error message through `build.rs`'s `slint_build::compile()`:

```
C:\...\slint-test\ui\app-window.slint:7: Cannot find image file https://slint.dev/logo/slint-logo-full-light.svg
```

It isn't clear why running the example in SlintPad and locally should differ in behavior. If local builds have a limitation regarding URLs, the message should reflect this and lead the developer on their way to understand the reasons and background better.

---

Also, the error is silent in the preview window shown via "Show Preview" from the VS Code editor. It just shows:

![silent-error](https://github.com/user-attachments/assets/18168d38-dd2b-4cd8-9a05-7c475e801373)

Whereas SlintPad shows:

![working](https://github.com/user-attachments/assets/bcd716f6-8f7c-4e08-9184-e844849bdcab)

**Comment by Enyium** [comment]
@ogoffart: The console showing an error message, but the preview window not, isn't a bug?!

**Comment by hunger** [comment]
Yes, Slintpad uses the interpreter internally. No, that has nothing to do with Slintpad being able to load images from URLs:-)

Slintpad runs WASm code, and WASM has no access to any filesystem at all. So we needed to hack something there to load get any data into the interpreter... that hack forwards load requests to the browser and browsers happen to be really good at loading things from the web. It's more an implementation detail than a planned feature ;-)

**Comment by Enyium** [comment]
But don't you think it's confusing to first see `@image-url("https://...` in the docs, and then not being able to make use of it? I think this should be mentioned in the docs and in the console error message. And the local preview should show the error message also.

**Comment by hunger** [comment]
I agree: This is confusing. Most of the time its a relative path, so it is hard to tell whether its a URL or a file path, so we do get away with it rather often.

But yes, that is a problem. The preview and Slintpad should be as close as possible to the "normal" compiler.
